primitives/Cargo.toml

```diff
@@ -27,5 +27,10 @@ tracing-test = { version = "0.2.1", git = "https://github.com/nathanwhit/tracing
 
 [features]
 default = ["std", "prometheus"]
-std = ["sp-core/std", "sp-arithmetic/std"]
+std = [
+    "sp-core/std",
+    "sp-arithmetic/std",
+    "sp-runtime-interface/std",
+    "parity-scale-codec/std",
```

Comment:
It's me being annoying again. I don't like including features unnecessarily. Are these two needed? I think it compiles with just `'primitives/std'`.